projects/plugins/jetpack/class.jetpack-gutenberg.php

```diff
@@ -209,45 +210,13 @@ public static function set_extension_unavailable( $slug, $reason, $details = arr
 	}
 
 	/**
-	 * Set up a list of allowed block editor extensions
+	 * Used to initialize the class, no longer in use.
 	 *
 	 * @return void
+	 * @deprecated $$next-version$$ No longer needed.
 	 */
 	public static function init() {
-		if ( ! self::should_load() ) {
-			return;
-		}
-
-		/**
-		 * Filter the list of block editor extensions that are available through Jetpack.
-		 *
-		 * @since 7.0.0
-		 *
-		 * @param array
-		 */
-		self::$extensions = apply_filters( 'jetpack_set_available_extensions', self::get_available_extensions() );
-
-		/**
-		 * Filter the list of block editor plugins that are available through Jetpack.
-		 *
-		 * @deprecated 7.0.0 Use jetpack_set_available_extensions instead
-		 *
-		 * @since 6.8.0
-		 *
-		 * @param array
-		 */
-		self::$extensions = apply_filters( 'jetpack_set_available_blocks', self::$extensions );
-
-		/**
-		 * Filter the list of block editor plugins that are available through Jetpack.
-		 *
-		 * @deprecated 7.0.0 Use jetpack_set_available_extensions instead
-		 *
-		 * @since 6.9.0
-		 *
-		 * @param array
-		 */
-		self::$extensions = apply_filters( 'jetpack_set_available_plugins', self::$extensions );
```

Comment:
There should be a change entry mentioning the removal of these deprecated filters (which would be of "major" significance, not that it matters since Jetpack doesn't use semver).

You could edit the existing entry or make a second one. Either is fine.

Reply:
Good point, [fixed](https://github.com/Automattic/jetpack/pull/30875/commits/41277236ea8e3d270d55463f8aba282b3836e436).